spec/controllers/miq_policy_controller/rsop_spec.rb

```diff
@@ -0,0 +1,28 @@
+include UiConstants
```

Comment:
Do we need this included globally?  @chrisarcand didn't you kill one of these recently?